src/Microsoft.Graph/Models/Generated/ChatMessageMention.cs

```diff
@@ -19,8 +19,7 @@ namespace Microsoft.Graph
     /// The type ChatMessageMention.
     /// </summary>
     [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
-    [JsonConverter(typeof(DerivedTypeConverter))]
-    public partial class ChatMessageMention
+    public partial class ChatMessageMention : Entity
```

Comment:
>: Entity [](start = 44, length = 8)

@MIchaelMainer, I don't see any change for chatMessageMention in metadata diff, what is the source of this change?

Reply:
Very good catch. This is really strange and requires investigation. A base type was applied after we captured the metadata, before we generated the 'clean' metadata. Interesting that it is being added as FQN without the alias..